src/webgpu/api/operation/reflection.spec.ts

```diff
@@ -0,0 +1,124 @@
+export const description = `
+Tests that object attributes which reflect the object's creation properties are properly set.
+`;
+
+import { makeTestGroup } from '../../../common/framework/test_group.js';
+import { GPUTest } from '../../gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('buffer_reflection_attributes')
+  .desc(`For every buffer attribute, the corresponding descriptor value is carried over.`)
+  .params(u =>
+    u.beginSubcases().combine('descriptor', [
```

Comment:
optionally:
```
  .paramsSubcasesOnly(u =>
    u.combine('descriptor', [
```

Reply:
Done! Going to wait till https://chromium-review.googlesource.com/c/chromium/src/+/3687793 lands to verify the test before landing, though.